lib/src/components/NxToast/__tests__/NxToastContainer.test.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React, { useState } from 'react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+import { screen, render, fireEvent } from '@testing-library/react';
+
+import userEvent from '@testing-library/user-event';
+
+import {reject, propEq } from 'ramda';
+
+import useToggle from '../../../util/useToggle';
+
+import NxToastContainer from '../NxToastContainer';
+import { NxToastContainerProps } from '../types';
+import NxToast from '../NxToast';
+import { NxErrorAlert } from '../../NxAlert/NxAlert';
+
+describe('NxToastContainer', function() {
+  const simpleProps: NxToastContainerProps = {
+    children: []
+  };
+
+  const quickRender = rtlRender<NxToastContainerProps>(NxToastContainer, simpleProps),
+      renderEl = rtlRenderElement(NxToastContainer, simpleProps);
+
+  it('renders a <div> with a class of nx-toast__wrapper', function() {
+    const component = quickRender();
+    const toastWrapper = component.container.children[0];
+
+    expect(toastWrapper.nodeName).toEqual('DIV');
+    expect(toastWrapper).toHaveClass('nx-toast__wrapper');
+  });
+
+  it('contains a<div> with a class of nx-toast__container', function() {
```

Comment:
I mentioned in `NxToast.test.tsx` that these sorts of things are probably implementation details that we don't want to test.  `NxToastContainer`'s tests are a perfect example of that.  The fact that we need two wrapper/container elements here is an implementation detail and might still change depending on https://github.com/sonatype/sonatype-react-shared-components/pull/688/files#r953796278 .  It shouldn't be asserted in the test

Reply:
> I mentioned in `NxToast.test.tsx` that these sorts of things are probably implementation details that we don't want to test. `NxToastContainer`'s tests are a perfect example of that. The fact that we need two wrapper/container elements here is an implementation detail and might still change depending on https://github.com/sonatype/sonatype-react-shared-components/pull/688/files#r953796278 . It shouldn't be asserted in the test

implentation details in tests have been removed here: [124d31a](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/124d31a171f3c37e46b17f1a6acb9e74d964e45d)